.gitignore

```diff
@@ -1,3 +1,7 @@
+.env
```

Comment:
I think we should make a sample `.env.example` that lays out the configs. The user will then copy paste the file, fill up the blank and rename the file to `.env`. It's more user friendly compared to the user having to figure out all the configs from the source code

Reply:
@imranariffin

Reply:
Agreed

Reply:
Done fb540d9